assets/javascripts/discourse/templates/knowledge-explorer-topic-link.hbr

```diff
@@ -0,0 +1 @@
+<a data-topic-id="{{topic.id}}" class="knowledge-explorer-topic-link">{{topic.title}}</a>
```

Comment:
I might be missing something, but did you look into using the `topic-link` helper? 

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/app/helpers/topic-link.js 

(It should be quite similar to the `category-link` helper used in the other raw template in this PR.

Reply:
Nevermind my earlier comment (the title doesn't load the regular topic, it triggers the custom `selectTopic` action.)